docs/reference/stdlib.md

```diff
@@ -322,30 +335,293 @@ assert a * 2 == b
 assert b / 2 == a
 ```
 
-The following methods are available for a `Duration` object:
+The following methods are available for a Duration:
 
-`getDays()`, `toDays()`
+`toDays() -> long`
```

Comment:
Getter should be preferred because they can be accessed also as attributes

Reply:
The getters like `getMillis()` are confusing because it's not clear if you're getting the millisecond representation of the whole duration or just the millisecond *component*, which is how it works in [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMilliseconds) and [Python](https://docs.python.org/3/library/datetime.html#datetime.datetime.second). Whereas `toMillis()` is clear that it's the former

Reply:
I added them back in a separate note so that it's clear we haven't removed them.

Reply:
Okey